controllers/helper.go

```diff
@@ -78,3 +79,12 @@ func containsString(slice []string, s string) bool {
 	}
 	return false
 }
+
+func checkCRD(crds crdv1.CustomResourceDefinitionList, api string) bool {
```

Comment:
what about `isCrdInstalled`

Reply:
change to `inCrdList()`, think `isCrdInstalled` is not correct, related the function is only compared an already fetched list. Was think to rename to `isCrdInstalled`, but this means to reference the `DatabaseReconciler` outside of the `database_controller.go`. Don't know if we want to limit on this, but was not feel right for me.